src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactoryTest.java

```diff
@@ -170,6 +176,7 @@ public void createWithDocumentSetFilters() throws IOException, DocumentSelectorE
         BundleConfiguration configuration = new BundleConfiguration(
             "Bundle title",
             "filename.pdf",
+                "FL-FRM-GOR-ENG-12345",
```

Comment:
it seems like this is repeated alot, many have a final variable and use it. not sure what the term was but i believe it was called magic number or something.